workspace/bus.ts

```diff
@@ -0,0 +1,5 @@
+export class Bus {
+    constructor(public line: string, public timeToArrival: string, public destination: string) {
+
+    }
```

Comment:
I would personally have these as `constuctor(<some args>) { }` to avoid whitespace.